### ISSUE TITLE
finetuning from mobilenet

### ISSUE THREAD

**Issue by ouceduxzk** [ildoonet/tf-pose-estimation#31]
Hi, I find that finetuning with mobilenet is very hard to converge, the training loss goes down to ~2000, while the validation loss is ~4000 with lr 0.0005, RMSProp, did you tried with other optimizers and learning rates?

Thanks

**Comment by ildoonet** [comment]
@ouceduxzk Yes, I tried others. But RMSProp gave me the best result. 

What kind of finetuning are you trying?

**Comment by zengxiang0217** [comment]
I had the same problem. Try to set lr to 0.01, it works for me.

**Comment by ouceduxzk** [comment]
@ildoonet  sorry for the late reply, just come back from holiday, I tried to train a workable model like you did by finetuning on mobilenet

**Comment by ouceduxzk** [comment]
@zengxiang0217  did you keep lr as a constant or you have lr decay policy, it seems such a high lr is not stable for training
